Makefile

```diff
@@ -128,15 +128,12 @@ endif
 
 XGO := $(strip $(if $(XGOOS),GOOS=$(XGOOS)) $(if $(XGOARCH),GOARCH=$(XGOARCH)) $(if $(XHOST_TRIPLE),CC=$(CC_PATH) CXX=$(CXX_PATH)) $(GO))
 
-.DEFAULT_GOAL := all
-.PHONY: all
-all: build test lint
```

Comment:
ok for DEFAULT_GOAL in light of the rules in common.mk

I still think that the PHONY spec is not needed though... oh well.